android/app/src/main/java/com/chorestory/templates/AccountResponse.java

```diff
@@ -15,7 +15,7 @@ public Data getData() {
         String email;
         Integer id;
         String name;
-        String clanName;
+        String clan_name;
```

Comment:
Very important

Reply:
retrofit serializes that and looks at the response for that exact word
so clanName doesnt work since you return clan_name

Reply:
can we leave a comment in the code